functions/Set-DbaAgentAlert.ps1

```diff
@@ -149,16 +150,16 @@ function Set-DbaAgentAlert {
             #endregion alert options
 
             # Execute
-            if ($PSCmdlet.ShouldProcess($SqlInstance, "Changing the alert $a")) {
+            if ($PSCmdlet.ShouldProcess($SqlInstance, "Committing changes for alert $a")) {
                 try {
-                    Write-Message -Message "Changing the alert" -Level Verbose
+                    Write-Message -Message "Committing changes for alert $a" -Level Verbose
 
                     # Change the alert
                     $currentAlert.Alter()
                 } catch {
                     Stop-Function -Message "Something went wrong changing the alert" -ErrorRecord $_ -Target $instance -Continue
                 }
-                Get-DbaAgentAlert -SqlInstance $server | Where-Object Name -eq $currentAlert.name
+                Get-DbaAgentAlert -SqlInstance $server | Where-Object Name -EQ $currentAlert.name
```

Comment:
Even if not the correct case, we use `-eq` with `Where-Object` for better readability.

Reply:
Sorry, don't follow - do you want me to change here something?

Reply:
Gotcha. I caught it in the other command, but this was changed by the formatted and I missed it to change it back to `-eq`